### PR TITLE
Issue 4962 - Fix various UI bugs - Database and Backups

### PR DESCRIPTION
**Pull request by droideck**
Description:

Bug 1751280 - [RFE] Cockpit : Provide the access path to the exported suffix
Bug 1861805 - 389-ds-base: Accepting nsslapd-db-checkpoint-interval values in negative.
Bug 1926516 - Cannot load the DB with suffix with characters escaped by a backslash.
Bug 1986388 - Cockpit: "Manage backups" list is empty if dirsrv service is stopped

Related: https://github.com/389ds/389-ds-base/issues/5001

Reviewed by: ?

### CODE REVIEW

**Comment by jchapma** [comment]
Hi @droideck 

Re: Bug 1926516

**Comment by droideck** [comment]
@jchapma hmm, it's weird... Works on 1minutetip machine for me. If you're applying the PR locally, have you installed the python changes too? (pip3 install src/lib389)

**Comment by jchapma** [comment]
> have you installed the python changes too? (pip3 install src/lib389)

@droideck 
Ye, Im applying it locally. I have not installed the python changes, let me try that. Thanks.

**Comment by jchapma** [comment]
> > have you installed the python changes too? (pip3 install src/lib389)
> 
> @droideck That solved the issue, you have my ack.

**Review by jchapma — APPROVED** [review]

File: src/lib389/lib389/cli_conf/backend.py [hunk 1]
```diff
@@ -94,7 +95,7 @@ def _search_backend_dn(inst, be_name):
         cn = ensure_str(be.get_attr_val('cn')).lower()
         suffix = ensure_str(be.get_attr_val('nsslapd-suffix')).lower()
         del_be_name = be_name.lower()
-        if cn == del_be_name or suffix == del_be_name:
+        if cn == del_be_name or str2dn(suffix) == str2dn(del_be_name):
```
Comment (mreynolds389) [inline src/lib389/lib389/cli_conf/backend.py:98]: Should really use "===" instead of "=="

**Comment by mreynolds389** [comment]
Besides that one function using the wrong comparator, LGTM

File: src/lib389/lib389/cli_conf/backend.py (hunk 1 above)
Comment (droideck) [reply src/lib389/lib389/cli_conf/backend.py:98]: Fix me if I'm wrong, but I think there's no such thing in Python...

**Comment by droideck** [comment]
But I found that the Instance creation was failing with the escape character suffix.
It is fixed now.

This was the change:

``` 
diff --git a/src/cockpit/389-console/src/dsModals.jsx b/src/cockpit/389-console/src/dsModals.jsx
index ad67e76937..069bf5b90f 100644
--- a/src/cockpit/389-console/src/dsModals.jsx
+++ b/src/cockpit/389-console/src/dsModals.jsx
@@ -322,7 +322,7 @@ export class CreateInstanceModal extends React.Component {
                                              */
                                             const cmd = [
                                                 '/bin/sh', '-c',
-                                                '/usr/bin/echo -e "' + setup_inf + '" >> ' + setup_file
+                                                '/usr/bin/echo -e \'' + setup_inf + '\' >> ' + setup_file
                                             ];
 
                                             // Do not log inf file as it contains the DM password
@@ -357,7 +357,7 @@ export class CreateInstanceModal extends React.Component {
                                                                     });
                                                                     addNotification(
                                                                         "error",
-                                                                        `${errMsg}`
+                                                                        `${errMsg.desc}`
                                                                     );
                                                                 })
```

File: src/lib389/lib389/cli_conf/backend.py (hunk 1 above)
Comment (mreynolds389) [reply src/lib389/lib389/cli_conf/backend.py:98]: Ummm, yeah you're right haha

**Comment by mreynolds389** [comment]
ack

**Pull request merged**